src/poprox_recommender/data/mind.py

```diff
@@ -0,0 +1,189 @@
+"""
+Support for loading MIND_ data for evaluation.
+
+.. _MIND: https://msnews.github.io/
+"""
+
+# pyright: basic
+import json
+import logging
+import zipfile
+from datetime import datetime, timezone
+from typing import Any, List, NamedTuple, Optional
+from uuid import UUID, uuid4
+
+import pandas as pd
+from pydantic import BaseModel
+from tqdm import tqdm
+
+from poprox_recommender.paths import project_root
+
+logger = logging.getLogger(__name__)
+
+
+class MindTables(NamedTuple):
+    """
+    News and behavior data loaded from MIND (raw data frames).
+    """
+
+    news: pd.DataFrame
+    behaviors: pd.DataFrame
+
+
+class MindData(NamedTuple):
```

Comment:
I might use Pydantic here for consistency but no objection to moving forward with this as is

Reply:
This is an intermediate class, it's going away in the next PR :)